README.md

```diff
@@ -15,6 +15,8 @@ See build(7), config(8), [FreeBSD handbook on building userland](https://docs.fr
 For information on the CPU architectures and platforms supported by FreeBSD, see the [FreeBSD
 website's Platforms page](https://www.freebsd.org/platforms/).
 
+Official FreeBSD bootable images on our [releases page](https://download.freebsd.org/ftp/releases/ISO-IMAGES/) to get yourself one. 
```

Comment:
I don't think that the other parts of this file use 'our' or similar 1st-person plural pronouns.
`For official FreeBSD bootable images, see our [release page](...)`
might be better?

Reply:
Is it an issue with using "our"?

Reply:
Yes. It's an informal style that doesn't match the rest of the doc.